README.md

```diff
@@ -2,7 +2,191 @@
+We're a friendly community and we would love to have more collaborators!
+
+The Wiki page
+[Getting started with contributing](https://github.com/metamath/set.mm/wiki/Getting-started-with-contributing) will walk you the process of contributing
+a final proof once you have one.
+Basically, you'll create a "pull request" on GitHub to request
+merging of some change(s).
+**Warning**: Changes are made to the **develop** branch.
+
+## What are the requirements for acceptance (merging)?
+
+A pull request (proposed change) *must* pass all verifiers before
+it will be merged (accepted). This even enforces some style rules on
+some databases, e.g., we want a name and date for each new theorem.
+
+To be merged, a pull request must be approved by a different existing
+contributor (someone who's already had some previous contributions accepted).
+You can approve a change by viewing the pull request, selecting
+the tab "Files Changed", clicking on the "Review Changes" button,
+clicking on "Approve", and submitting the review.
+You can also approve by using the comment "+1".
+Feel free to approve something others have approved, that makes it clear
+there's no serious issue.
+
+Once a pull request is approved (other than the person
+originally proposing the change), any maintainer can merge it,
+including the approver.
+A maintainer is a member or owner of the Metamath GitHub organization.
+
+We-generally tend to have a bias towards "merge this now" with separate
+follow-up fixes for minor issues (e.g., to use a better name). That reduces the
+risk of losing good ideas, or a lot of rework, due to some minor
+blemish like an odd name. It's a pain to fix "merge conflicts"
+because differences have accumulated over time, and we'd rather avoid
+merge conflicts if there's no good reason to have them.
+
+We encourage changes to be smaller, focusing on a single logical idea.
+That makes changes much easier to review.
+It also makes it easy to accept some changes while not accepting others.
+
+Many contributors have or create a personal "mathbox" - a sandbox of work that
+is visible to others. Changes to ones' own mathbox still go through a
+review, but usually it's cursory - we generally just want to ensure that those
+changes don't interfere with or confuse others.
+Mathbox changes must still pass all verifiers, though, and that is
+a very rigorous check.
+
+Some areas may have "area maintainers" - changes to those areas
+must generally also be approved by at least one of the area maintainer(s).
+Similarly, mathboxes should normally only be changed by the owner of the
+mathbox (unless it's a general improvement that applies across the database).
+We don't currently have a list of area maintainers; please propose
+a change to this README (here) if you want to be an area maintainer.
+
+## Credits and a memorium
+
+Our sincere thanks to *everyone* who has contributed to this work.
+
+In addition, this entire collection is dedicated to the memory of
```

Comment:
Well done. We can also think about writing something at the start of Norm's mathbox (similar to the one we have for Alan Sare), but this doesn't have to be done all at once.

Reply:
Thanks! Since Norm created all this, I think it's reasonable as a special case to put it here in the README. But you're right, it'd be good to say something in his mathbox too... but let's do that separately. In fact, if this is in the README, I think there's no hurry.